### PR TITLE
Added Advice Generator

### LINKED ISSUE #2097: Random Advice App
**Describe the project you want to add with tech stack**
Generates life motivational advices on clicking the button

**Screenshots (optional)**

image

image

### PR DESCRIPTION
**Pull request by PoulavBhowmick03**
# Title and Issue number 
Random Advice App #2097

Close #2097 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

https://github.com/pranjay-poddar/Dev-Geeks/assets/133862694/8cfb03cd-b798-4f63-96f9-ffac59fa615c

### CODE REVIEW

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 please merge this

File: .vscode/settings.json
Comment (MohitGupta121) [inline .vscode/settings.json:1]: Why you again changing this.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (PoulavBhowmick03) [reply .vscode/settings.json:1]: my bad, i was using the live server so it automatically changed

**Comment by PoulavBhowmick03** [comment]
@MohitGupta121 Removed the file

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**